### PR TITLE
Updates server install documentation

### PR DESCRIPTION
**Pull request by heartsucker**
Related:
https://github.com/freedomofpress/securedrop/issues/1249
https://github.com/freedomofpress/securedrop/issues/1244

### CODE REVIEW

**Comment by conorsch** [comment]
These are mostly positive changes, particularly the note pertaining to #1244. The recommendation to manually set the DNS servers seems redundant, however, given that the Ansible playbooks already handle that setup in `roles/common/tasks/harden_dns.yml`. Admittedly the playbook only handles setting one DNS server, rather than two, but we have an issue open to track that as well:  #1114.

The encouragement to read the whole install guide, regardless of familiarity with installing Ubuntu, is frank and potentially quite useful to new admins, so thank you for that contribution.

**Comment by ehartsuyker** [comment]
@conorsch When running `securedrop-prod.yml`, `install_fpf_repo` is the first role/task run. It uses a DNS record (`apt.freedom.press`), so if the admin didn't correctly set the DNS records at install time, this will fail in a non-obvious way.

I can update the docs here to remove that bit about DNS and open a PR to shuffle the playbooks so that the DNS is set first. If y'all are so inclined.

And actually https://github.com/freedomofpress/securedrop/issues/1247 is related in some sense in that things using TLS depend on `ntp`. Maybe these together are worth an issues or a partial refactor of `ansible` so that fresh install don't hit these little hiccups like I did.

**Comment by garrettr** [comment]
> open a PR to shuffle the playbooks so that the DNS is set first

This seems reasonable to me, and removes what is currently an implicit dependency in the Ubuntu installation process (that you follow our documentation and set up DNS servers when installing Ubuntu). @conorsch What do you think?

The other option is to configure the network firewall to act as a DNS server, which would mean Ubuntu's default behavior of using the gateway for DNS would lead to correct DNS resolution. James and I debated doing that a while back, unfortunately I cannot recall exactly why we decided this was the way to go.

**Comment by conorsch** [comment]
@garrettr Optimizing the Ansible playbooks to be more resilient is always a good thing, so I'm in favor of making changes here. Rather than patch the playbooks in master, however, we should work on getting develop merged in for the new release and make changes on top of that.

File: docs/development/getting_started.rst [hunk 1]
```diff
@@ -327,7 +327,7 @@ Using Tor Browser with the development environment
 We strongly encourage sources to use the Tor Browser when they access
 the Source Interface. Tor Browser is the easiest way for the average
 person to use Tor without making potentially catastrophic mistakes,
-makes disable Javascript easy via the handy NoScript icon in the
+makes disabling Javascript easy via the handy NoScript icon in the
```
Comment (conorsch) [inline docs/development/getting_started.rst]: :+1:

File: docs/hardware.rst [hunk 2]
```diff
@@ -224,6 +224,11 @@ insert the cards into the NUC before it can be used. We recommend:
 	  updating the BIOS according to `these instructions
 	  <http://arstechnica.com/gadgets/2014/02/new-intel-nuc-bios-update-fixes-steamos-other-linux-booting-problems/>`__.
 
+.. note:: Some BIOS versions on the NUC will cause the server to
+	  `brick itself <https://communities.intel.com/message/359708>`__ if
+	  the device attempts to suspend. Some suggestions include disabling
+	  suspend in the BIOS as well as OS options like "wake on LAN".
+
```
Comment (conorsch) [inline docs/hardware.rst]: Great advice, see #1244 for context.

**Comment by conorsch** [comment]
There are multiple changes in this PR:

1. documentation potential "bricking" of NUCs (see #1244)
2. fix minor typo in Tor Browser Bundle description
3. refactor docs flow for OS installation

1 and 2 are easy to review and should be merged straightaway. 3 requires more careful review, but in general the tone is frank and accessible, and definitely improves the project documentation. Note that the **Minor Admin Tasks** section explains steps for rectifying system state that can be automated away via Ansible, but isn't yet—see #1249.

+1 for merge.

**Comment by conorsch** [comment]
Also includes FDE warning discussed in #1266, and therefore closes #1265.

**Comment by garrettr** [comment]
Gonna trust @conorsch's +1 review for this one and merge.

**Pull request merged**